src/pages/dashboard/registration.tsx

```diff
@@ -841,7 +841,10 @@ function Registration() {
   };
 
   const previousStep = () => {
-    if (step === steps.personalInformation) return;
+    if (step === steps.personalInformation) {
+      history.push('/');
```

Comment:
As @dellisd mentioned. 
```suggestion
      history.push('/dashboard');
```